.wakemanifest

```diff
@@ -0,0 +1,69 @@
+src/types/types.wake
+src/src.wake
+src/optimizer/optimizer.wake
+src/dst/dst.wake
+src/wcl/wcl.wake
+src/json/json.wake
+src/util/util.wake
+src/parser/parser.wake
+src/wakefs/wakefs.wake
+src/compat/compat.wake
+src/runtime/runtime.wake
+src/job_cache/job-cache.wake
+build.wake
+extensions/vscode/vscode.wake
+tests/tests.wake
+rust/log_viewer/build.wake
+share/wake/lib/gcc_wake/gcc.wake
+share/wake/lib/gcc_wake/pkgconfig.wake
+share/wake/lib/rust_wake/cargo.wake
+share/wake/lib/system/job.wake
+share/wake/lib/system/plan_scorer.wake
+share/wake/lib/system/sources.wake
+share/wake/lib/system/environment.wake
+share/wake/lib/system/io.wake
+share/wake/lib/system/path.wake
+share/wake/lib/nothing/nothing.wake
+share/wake/lib/core/list.wake
+share/wake/lib/core/order.wake
+share/wake/lib/core/regexp.wake
+share/wake/lib/core/print.wake
+share/wake/lib/core/tuple.wake
+share/wake/lib/core/json.wake
+share/wake/lib/core/option.wake
+share/wake/lib/core/boolean.wake
+share/wake/lib/core/vector.wake
+share/wake/lib/core/double.wake
+share/wake/lib/core/integer.wake
+share/wake/lib/core/tree.wake
+share/wake/lib/core/types.wake
+share/wake/lib/core/result.wake
+share/wake/lib/core/string.wake
+share/wake/lib/core/map.wake
+share/wake/lib/core/syntax.wake
```

Comment:
I think we should omit these from the wake repo as they will also be omitted in other repos. Wakes shared library finding technique should work inside the wake repo as long as wake executables sit inside of the bin directory.

Reply:
When wake notices the stdlib in the current workspace it chooses to use that instead (by falling back to the process for finding all wake files in the workspace). Since we replace that mechanism with the manifest and we have the stdlib in our workspace, we have to use the manifest to locate these files

Reply:
I think we should instead have wake always use `{execpath}/../share/lib` as the standard lib personally.

Reply:
(not in this change to be clear, just how I wish it worked)